package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "io-eslint-rules",
```

Comment:
why the package and repo names diverge?

Reply:
The name of the package must be `@pagopa/eslint-config` to work properly. See [this](https://eslint.org/docs/developer-guide/shareable-configs#sharing-multiple-configs) ESLint documentation reference. If we want we can rename the repo to `eslint-config`. Fixed in 7dd1a75